src/E1_process_results.py

```diff
@@ -310,12 +416,26 @@ def get_flow(settings, bus, dict_asset, bus_name, direction):
 
 
 def add_info_flows(settings, dict_asset, flow):
-    """
+    r"""
+    Adds `flow` and total flow amongst other information to `dict_asset`.
+
+    Parameters
+    ----------
+    settings : dict
+        Contains simulation settings from `simulation_settings.csv` with
+        additional information like the amount of time steps simulated in the
+        optimization ('periods').
```

Comment:
Indent problem?

Reply:
No.. or I don't know what you mean?

Reply:
Oh maybe it was indent problem linked to the size of my screen!